lib/aida/skills/decision_tree.ex

```diff
@@ -152,12 +152,12 @@ defmodule Aida.Skill.DecisionTree do
   end
 
   defimpl Aida.Skill, for: __MODULE__ do
-    def init(skill, _bot) do
-      skill
-    end
+    def init(skill, _bot), do: skill
+
+    def wake_up(_skill, _bot, _data), do: :ok
```

Comment:
May be best not to change this stuff in the same commit of a feature, so the diffs get easier to read

Reply:
👌